src/components/text-editor/__internal__/toolbar/toolbar.component.tsx

```diff
@@ -119,19 +116,16 @@ const Toolbar = ({
           controlRefs.current[focusIndex + 1].current?.focus();
           setFocusIndex(focusIndex + 1);
         }
-        setTabbable(false);
       }
     },
     [controlRefs, focusIndex, handleBlockType, handleInlineStyleChange]
   );
 
-  useEffect(() => {
-    isFirstRender.current = false;
-  }, []);
-
   useEffect(() => {
     if (canFocus && focusIndex === 0) {
       setTabbable(true);
+    } else if (focusIndex !== 0) {
```

Comment:
Wouldn't just using an `else` work here?

Reply:
When I tried just `else` on initial render it would change `tabbale` to false meaning the toolbar could be focused from keyboard nav if you hadn’t typed something in the text editor first. The `else if` stopped this from happening.

Reply:
Yeah I prefer it as it was originally as well tbh, but I felt Maciek had moved it into an effect hook for a reason so just wanted it consolidated in one place rather than in handler and then a hook etc.

Reply:
Reverted back to how it was.